lib/spack/spack/cmd/style.py

```diff
@@ -343,10 +344,15 @@ def prefix_relative(path):
                 run_function, required = tools[tool_name]
                 print_tool_header(tool_name)
 
-                cmd = which(tool_name, required=required)
-                if not cmd:
-                    color.cprint("  @y{%s not in PATH, skipped}" % tool_name)
-                    continue
+                # Bootstrap tools so we don't need to require install
+                with spack.bootstrap.ensure_bootstrap_configuration():
+                    spec = spack.spec.Spec("py-%s" % tool_name)
+                    spec.concretize()
```

Comment:
`bootstrap.get_executable()` will concretize if it doesn't find the executable, so you can avoid paying the cost of concretization in the fast case (i.e. it's installed already or in the environment).